helm-charts/fluent-bit_values.yaml

```diff
@@ -0,0 +1,272 @@
+# Minikube stores its logs in a separate directory.
+# Enable if you install the chart in minikube.
+on_minikube: false
+
+image:
+  fluent_bit:
+    repository: fluent/fluent-bit
+    tag: 1.3.2
+  pullPolicy: Always
+
+testFramework:
+  image: "dduportal/bats"
+  tag: "0.4.0"
+
+nameOverride: ""
+fullnameOverride: ""
+
+# When enabled, exposes json and prometheus metrics on {{ .Release.Name }}-metrics service
+metrics:
+  enabled: false
+  service:
+    # labels:
+    #   key: value
+    annotations: {}
+    # In order for Prometheus to consume metrics automatically use the following annotations:
+    # prometheus.io/path: "/api/v1/metrics/prometheus"
+    # prometheus.io/port: "2020"
+    # prometheus.io/scrape: "true"
+    port: 2020
+    type: ClusterIP
+  serviceMonitor:
+    enabled: false
+    additionalLabels: {}
+    # namespace: monitoring
+    # interval: 30s
+    # scrapeTimeout: 10s
+
+# When enabled, fluent-bit will keep track of tailing offsets across pod restarts.
+trackOffsets: false
+
+## PriorityClassName
+## Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
+priorityClassName: ""
+
+backend:
+  type: es
+  es:
+    host: "elasticsearch.example.com"
```

Comment:
Are these all dummy values or does this `host` field need to be updated?

Reply:
@gigawhitlocks The host is the only thing that needs to be updated and its done by the provisioner code.